src/test/java/io/cdap/plugin/switchcase/route/BasicRoutingSwitchTest.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.switchcase.route;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.etl.api.InvalidEntry;
+import io.cdap.cdap.etl.api.SplitterTransform;
+import io.cdap.cdap.etl.mock.common.MockMultiOutputEmitter;
+import io.cdap.cdap.etl.mock.transform.MockTransformContext;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class BasicRoutingSwitchTest extends RoutingSwitchTest {
```

Comment:
Public class comment.

Reply:
fixed. Wonder why checkstyle doesn't cause this to fail.